nc_time_axis/__init__.py

```diff
@@ -248,17 +248,21 @@ def convert(cls, value, unit, axis):
                 return value
             first_value = value
 
+        if not isinstance(first_value, CalendarDateTime):
+            msg = 'The value must be a number, a sequence of numbers or of ' \
+                  'type :class:`nc_time_axis.CalendarDateTime.'
```

Comment:
In fact this is only checking the type of the first value which should be a number or `CalendarDateTime` and not a sequence at all